packages/itwinui-css/backstop/tests/date-picker.html

```diff
@@ -2643,5 +2643,510 @@ <h2>Date range</h2>
         </div>
       </div>
     </section>
+
+    <h3>Some dates disabled</h2>
```

Comment:
```suggestion
    <h3>Some dates disabled</h3>
```

This should fix the font size issue.